package.json

```diff
@@ -1,7 +1,9 @@
 {
+  "name": "pymedphys",
   "private": true,
   "scripts": {
     "bootstrap": "yarn && yarn conda:forge && yarn dependencies && yarn nbstripout:install && yarn install:dev",
+    "lerna:bootstrap": "lerna bootstrap && pip install -e .",
```

Comment:
I added this back in, just so that we can use `lerna` if need be. We will need to zero in on one tool that works well, but I found `lerna` made the installation quite effortless.

So essentially `yarn lerna:boostrap` will do what just `yarn` used to do before I attempted to swap over to requirement files.

Reply:
@Centrus007

Reply:
@Centrus007